### PR TITLE
Scripts for TDR-2097 and TDR-2101

### PR DESCRIPTION
**Pull request by pashworth-tna**
Two scripts, One for enhancing metadata and one for add a MOCKA body and series

### CODE REVIEW

File: lambda/src/main/resources/db/migration/V57__ClosureMetadataUpdates.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+UPDATE "FileProperty" SET "PropertyGroup" = 'MandatoryClosure'  WHERE "Name" = 'ClosureType';
+UPDATE "FileProperty" SET "PropertyGroup" = 'MandatoryClosure'  WHERE "Name" = 'ClosurePeriod';
+UPDATE "FileProperty" SET "PropertyGroup" = 'MandatoryClosure', "Datatype" = 'boolean'  WHERE "Name" = 'DescriptionPublic';
+UPDATE "FileProperty" SET "PropertyGroup" = 'MandatoryClosure', "Datatype" = 'boolean'  WHERE "Name" = 'TitlePublic';
+UPDATE "FileProperty" SET "PropertyGroup" = 'OptionalClosure'  WHERE "Name" = 'ClosureStartDate';
+UPDATE "FileProperty" SET "PropertyGroup" = 'OptionalClosure'  WHERE "Name" = 'DescriptionAlternate';
+UPDATE "FileProperty" SET "PropertyGroup" = 'OptionalClosure'  WHERE "Name" = 'TitleAlternate';
+UPDATE "FileProperty" SET "PropertyGroup" = 'MandatoryClosure'  WHERE "Name" = 'FoiExemptionAsserted';
+UPDATE "FileProperty" SET "PropertyGroup" = 'MandatoryClosure'  WHERE "Name" = 'FoiExemptionCode';
+
```
Comment (TomJKing) [inline lambda/src/main/resources/db/migration/V57__ClosureMetadataUpdates.sql:16]: Unnecessary line break

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a couple of minor style comments.

File: lambda/src/main/resources/db/migration/V58__Add_new_MOCKA_dept.sql [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        (uuid_generate_v4(), 'MOCKA', 'MockA', 'TDR-MOCKA') RETURNING "BodyId" INTO bodyUuid;
+
+        -- Use the returned bodyUuid value for the series insert
+        INSERT INTO "Series" ("SeriesId", "BodyId", "Code", "Name", "Description") VALUES
+        (uuid_generate_v4(), bodyUuid, 'MOCKA 101', 'MOCKA 101', 'MOCKA 101');
+
+    END $$;
+
+--commit changes
+COMMIT;
```
[the remark this answers is not in the document]
Comment (pashworth-tna) [reply lambda/src/main/resources/db/migration/V58__Add_new_MOCKA_dept.sql]: done

File: lambda/src/main/resources/db/migration/V57__ClosureMetadataUpdates.sql (hunk 1 above)
Comment (pashworth-tna) [reply lambda/src/main/resources/db/migration/V57__ClosureMetadataUpdates.sql:16]: done

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**